test/test.html

```diff
@@ -0,0 +1,64 @@
+    }
+    const test1 = {"script_name":"chode droods","loot":[{"id":203,"name":"Grimy tarromin","count":36},{"id":203,"name":"Grimy tarromin","count":36},{"id":203,"name":"Grimy tarromin","count":36},{"id":203,"name":"Grimy tarromin","count":36},{"id":199,"name":"Grimy guam leaf","count":84},{"id":213,"name":"Grimy kwuarm","count":5},{"id":561,"name":"Nature rune","count":4},{"id":211,"name":"Grimy avantoe","count":20},{"id":217,"name":"Grimy dwarf weed","count":2},{"id":201,"name":"Grimy marrentill","count":44},{"id":563,"name":"Law rune","count":32},{"id":995,"name":"Coins","count":5420},{"id":985,"name":"Tooth half of key","count":2},{"id":207,"name":"Grimy ranarr weed","count":20},{"id":215,"name":"Grimy cadantine","count":9},{"id":9142,"name":"Mithril bolts","count":116},{"id":13471,"name":"Ensouled chaos druid head","count":8},{"id":205,"name":"Grimy harralander","count":24},{"id":209,"name":"Grimy irit leaf","count":20},{"id":2485,"name":"Grimy lantadyme","count":6},{"id":231,"name":"Snape grass","count":8},{"id":1621,"name":"Uncut emerald","count":2}],"runtime":101,"xp_earned":[{"skill":"strength","xp":"42,069"},{"skill":"hitpoints","xp":"11,022"}]};
+    const test2 = {"script_name":"seers agility","runtime":60,"xp_earned":[{"skill":"agility","xp":"126,585"}],"loot":[{"id":11849,"name":"Mark of grace","count":43}]};
+    const test3 = {"script_name":"too many skills","runtime":33,"xp_earned":[{"skill":"smithing","xp":"11,432"},{"skill":"herblore","xp":"1,500"},{"skill":"agility","xp":"126,585"},{"skill":"strength","xp":"42,069"},{"skill":"hitpoints","xp":"11,022"}],"loot":[{"id":11849,"name":"Mark of grace","count":43}]};
+    const test4 = {"script_name":"no loot","runtime":222,"xp_earned":[{"skill":"mining","xp":"1,111,432"},{"skill":"smithing","xp":"11,432"},{"skill":"herblore","xp":"1,500"},{"skill":"agility","xp":"126,585"},{"skill":"strength","xp":"42,069"},{"skill":"hitpoints","xp":"11,022"}]};
+    const test5 = {"script_name":"no xp","runtime":61,"loot":[{"id":995,"name":"Coins","count":322322},{"id":995,"name":"Coins","count":1500332},{"id":995,"name":"Coins","count":800},{"id":995,"name":"Coins","count":12350},{"id":995,"name":"Coins","count":5420},{"id":203,"name":"Grimy tarromin","count":36},{"id":199,"name":"Grimy guam leaf","count":84}]};
+    const test6 = {"script_name":"lol xd 420 funny", "runtime": 420, "loot":[{"id": 6055, "count": 420},{"id": 18777, "count": 1}],"xp_earned":[{"skill": "herblore", "xp":"420"}]};
+    
+    async function getImageAsync(testObj){
+      let json = await axios.post('/getImage', testObj, headers);
+      return json.data.body;
+    }
+
+    (async()=>{
+      let t1 = await getImageAsync(test1);
+      let t2 = await getImageAsync(test2);
+      let t3 = await getImageAsync(test3);
+      let t4 = await getImageAsync(test4);
+      let t5 = await getImageAsync(test5);
+      let t6 = await getImageAsync(test6);
+      const imgArray =[t1, t2, t3, t4, t5, t6];
```

Comment:
what if, test1 to test6 were all objects in an array (or object if names are important to you) from the beginning, then you could do a 
```
const imgArray = await promise.all(
  tests.forEach(a => getImageAsync(a))
);
```